Avoid duplicate declaration of prctl_mm_map:

In file included from ../src/basic/missing_prctl.h:4,
                 from ../src/basic/capability-util.c:16:
/usr/include/linux/prctl.h:134:8: error: redefinition of 'struct prctl_mm_map'
  134 | struct prctl_mm_map {
      |        ^~~~~~~~~~~~
In file included from ../src/basic/capability-util.c:6:
/usr/include/sys/prctl.h:88:8: note: originally defined here
   88 | struct prctl_mm_map {
      |        ^~~~~~~~~~~~

diff -rup missing_prctl.h.orig missing_prctl.h
--- a/src/basic/missing_prctl.h
+++ b/src/basic/missing_prctl.h
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
-#include <linux/prctl.h>
+#include <sys/prctl.h>
 
 /* 58319057b7847667f0c9585b9de0e8932b0fdb08 (4.3) */
 #ifndef PR_CAP_AMBIENT
